ausdex/inflation.py

```diff
@@ -82,7 +87,7 @@ def cpi_at(
 
         min_date = cpi_series.index.min()
         cpis = np.array(
-            cpi_series[np.searchsorted(cpi_series.index, date, side="right") - 1],
+            cpi_series[np.searchsorted(cpi_series.index.values, date, side="right") - 1],
```

Comment:
@rbturnbull in response to the request to go pandas > 1.3.2 This might be a breaking change here (for > 2.0 it changed how to access datetime indices. We could make function that does it one way for one, teh other way for another)